readthedocs/restapi/views/integrations.py

```diff
@@ -51,10 +55,18 @@ class WebhookMixin(object):
     renderer_classes = (JSONRenderer,)
     integration = None
     integration_type = None
+    invalid_payload_msg = 'Payload not valid'
```

Comment:
We may want to be more explicit with this. Like: "Payload's signature is not valid"